src/Utils/embedding_utils.py

```diff
@@ -3,15 +3,14 @@
 import time
 
 from langchain_community.vectorstores import FAISS
-from src.LLMService import LLMService
 from transformers import AutoTokenizer
 
 from Utils.file_utils import read_all_source_code_files
 
 logger = logging.getLogger(__name__)
 
 
-def generate_code_embeddings(llm_service: LLMService):
+def generate_code_embeddings(llm_service):
```

Comment:
shall we refactor this method signature to accept only `vector_service` and `embedding_llm`?